src/routes/store/index.svelte

```diff
@@ -1,18 +1,89 @@
 <script context="module">
+  import * as api from '@/utils/api.js';
+  const productLimit = 24;
+
   export async function preload() {
-    const res = await this.fetch('/store.json');
-    return {items: await res.json()};
+    const { pages, data } = await api.get(`/products?limit=${productLimit}`);
```

Comment:
Here is the point when it is really better to use `this.fetch` Sapper's wrapper instead of fetch API or node-fetch ([explained](https://sapper.svelte.dev/docs#this_fetch)). The main comment is in _@/utils/api.js_

Reply:
I'll be honest here, when I copied the code for `api.js`, I thought it was making use of `this.fetch`. Now that I look at it, I see that it doesn't. Moreover, I thought I was using that `store.json.js` frontend endpoint, but I now see that I'm not. So far it works and I don't know how far this will take us, and if changing any of this is really necessary. That server endpoint looks a little redundant to me. 

Anyway, bottomline: just stick with it, if we encounter issues, we will fix them at the spot.